README.md

```diff
@@ -77,10 +77,12 @@ For details on the `json` and `ldap` backends see the [documentation](https://ow
         -p 127.0.0.1:389:389 \
         -p 636:636 -d osixia/openldap
     ```
+1.  start a Redis server e.g. with docker:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[ordered-list-marker-value] Marker should be `2`, was `1`](https://app.codacy.com/gh/owncloud/ocis-reva/pullRequest?prid=4997850)